### PR TITLE
Add support for ignored files

### PR DESCRIPTION
**Pull request by mtancoigne**
This is directly dependent on https://github.com/julianrubisch/attractor/pull/92.

### CODE REVIEW

File: lib/attractor/calculators/js_calculator.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 module Attractor
   class JsCalculator < BaseCalculator
-    def initialize(file_prefix: '', minimum_churn_count: 3, start_ago: 365 * 5)
-      super(file_prefix: file_prefix, file_extension: '(js|jsx)', minimum_churn_count: minimum_churn_count, start_ago: start_ago)
+    def initialize(file_prefix: '', ignores: '', minimum_churn_count: 3, start_ago: 365 * 5)
+      super(file_prefix: file_prefix, ignores: ignores, file_extension: '(vue)', minimum_churn_count: minimum_churn_count, start_ago: start_ago)
```
Comment (julianrubisch) [inline lib/attractor/calculators/js_calculator.rb:6]: please leave the default file extension :-)

Comment (mtancoigne) [reply lib/attractor/calculators/js_calculator.rb:6]: Ooops... Sorry ;)

I'll re-push as soon as I get home.

**Comment by mtancoigne** [comment]
Sorry for the delay, it's done.

**Comment by julianrubisch** [comment]
Great! I'll release as 0.2 later today. Is that enough for you to update the attractor PR?

**Comment by julianrubisch** [comment]
@all-contributors add @mtancoigne for code

**Pull request merged**